### PR TITLE
Fix web SDK for backwards compat, bump ver numbers

### PR DESCRIPTION
**Pull request by msub2**

### CODE REVIEW

File: web/src/index.js [hunk 1]
```diff
@@ -1,24 +1,40 @@
...
         this.height = "100%";
         this.shadow = this.attachShadow({mode: 'open'});
     }
 
     connectedCallback() {
+        this.style.cursor = "pointer";
+        
         this.adSpace = this.getAttribute("adspace");
         this.creator = this.getAttribute("creator");
+        this.adFormat = this.hasAttribute("adformat") ? this.getAttribute("adformat") : this.adFormat;
```
Comment (limbofeather) [inline web/src/index.js:20]: Is `format` a reserved variable? Would like to start removing the word `ad` from across Zesty.

File: web/src/index.js [hunk 2]
```diff
@@ -1,24 +1,40 @@
...
         this.height = this.hasAttribute("height") ? this.getAttribute("height") : this.height;
         this.width = this.hasAttribute("width") ? this.getAttribute("width") : this.width;
 
-        async function loadAd(adSpace, creator, shadow, width, height) {
+        // Adjust height and width after getting initial values
+        let numMatch = /(\d+)/; // Will split to get an array of ["", num, suffix]
```
Comment (limbofeather) [inline web/src/index.js:25]: Could we wrap this functionality into a function with some docs?

**Review by limbofeather — APPROVED** [review]
Just a few comments but everything LGTM otherwise.

File: web/src/index.js (hunk 1 above)
Comment (msub2) [reply web/src/index.js:20]: Don't think it is. I'll address this in a separate PR that'll just go through and update naming.

**Pull request merged**